### PR TITLE
LastFM.DesktopScrobbler version 3.1.29

### PR DESCRIPTION
**Pull request by sanyer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16584)

### CODE REVIEW

File: manifests/l/LastFM/LastFMDesktopScrobbler/3.1.29/LastFM.LastFMDesktopScrobbler.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: LastFM.LastFMDesktopScrobbler
+PackageVersion: 3.1.29
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
```
Comment (OfficialEsco) [inline manifests/l/LastFM/LastFMDesktopScrobbler/3.1.29/LastFM.LastFMDesktopScrobbler.installer.yaml:6]:
```suggestion
  Architecture: x86
```

File: manifests/l/LastFM/LastFMDesktopScrobbler/3.1.29/LastFM.LastFMDesktopScrobbler.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  InstallerType: msi
+  InstallerUrl: https://cdn.last.fm/client/Win/LastFM_Scrobbler_3.1.29.msi
+  InstallerSha256: DF00345A5D09C7598541B19F20709C5B2CA531762304184ED89C2F1A0A28A8CA
+  ProductCode: '{B13709CB-85AE-4F45-BFF9-2CB2B7A78F83}'
```
Comment (OfficialEsco) [inline manifests/l/LastFM/LastFMDesktopScrobbler/3.1.29/LastFM.LastFMDesktopScrobbler.installer.yaml:10]:
```suggestion
  ProductCode: '{B13709CB-85AE-4F45-BFF9-2CB2B7A78F83}'
  Scope: machine
```

File: manifests/l/LastFM/LastFMDesktopScrobbler/3.1.29/LastFM.LastFMDesktopScrobbler.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+PackageIdentifier: LastFM.LastFMDesktopScrobbler
+PackageVersion: 3.1.29
+PackageLocale: en-US
+Publisher: LastFM
```
Comment (OfficialEsco) [inline manifests/l/LastFM/LastFMDesktopScrobbler/3.1.29/LastFM.LastFMDesktopScrobbler.locale.en-US.yaml:5]:
```suggestion
Publisher: Last.fm
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Please do these quick changes :)

File: manifests/l/LastFM/LastFMDesktopScrobbler/3.1.29/LastFM.LastFMDesktopScrobbler.locale.en-US.yaml (hunk 3 above)
Comment (sanyer) [reply manifests/l/LastFM/LastFMDesktopScrobbler/3.1.29/LastFM.LastFMDesktopScrobbler.locale.en-US.yaml:5]: this change will also require changes to PackageIdentifier and folders, right?

Comment (OfficialEsco) [reply manifests/l/LastFM/LastFMDesktopScrobbler/3.1.29/LastFM.LastFMDesktopScrobbler.locale.en-US.yaml:5]: Nope, the folder structure does not have to match, that would also cause another issue..

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**